py/function.go

```diff
@@ -26,7 +27,6 @@ type Function struct {
 	Name        string     // The __name__ attribute, a string object
 	Dict        StringDict // The __dict__ attribute, a dict or NULL
 	Weakreflist List       // List of weak references
-	Module      Object     // The __module__ attribute, can be anything
```

Comment:
naive question: why do we drop this field?
is it available now from `Ctx`?

Reply:
I am open to to add it back -- I just didn't see a use for it (and seemed unused before).  Now that `type Method struct` has a valid`*Module` set, module Methods now always have self set properly:

https://github.com/drew-512/gpython/blob/5ebaf66936cc1f96e34a348cf7066c0369d9f7c7/py/method.go#L236